Content.Server.Database/Migrations/Postgres/PostgresServerDbContextModelSnapshot.cs

```diff
@@ -231,17 +235,29 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("character varying(4096)")
                         .HasColumnName("message");
 
+                    b.Property<int>("NoteSeverity")
+                        .HasColumnType("integer")
+                        .HasColumnName("note_severity");
+
+                    b.Property<int>("NoteType")
+                        .HasColumnType("integer")
+                        .HasColumnName("note_type");
+
                     b.Property<Guid>("PlayerUserId")
                         .HasColumnType("uuid")
                         .HasColumnName("player_user_id");
 
+                    b.Property<TimeSpan>("PlaytimeAtNote")
+                        .HasColumnType("interval")
+                        .HasColumnName("playtime_at_note");
```

Comment:
This should be derived from their playtime.

Reply:
How? It's playtime *at the time of getting the note*

Reply:
I thought each entry was tracked but I realised it isn't.